### PR TITLE
account-service, in progress

### PR DESCRIPTION
**Pull request by thok54**

### CODE REVIEW

File: src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+public class MySqlAccountRepository implements AccountRepository {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Override
+    public List<Account> findAll() {
+        return jdbcTemplate.query("select * from ACCOUNTS", new RowMapper<Account>() {
+            @Override
+            public Account mapRow(ResultSet rs, int rowNum) throws SQLException {
```
Comment (jantpedraza) [inline src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java]: As commented in `bank-service`, this method is duplicated in this class 3 times. Code must not be duplicated unless there is a good reason for it.

File: src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+
+    @Override
+    public void store(Account account) {
+        String name = account.getName();
+        Float money = account.getMoney();
+        String iban = account.getIban();
+        jdbcTemplate.update(String.format(
+                "INSERT INTO ACCOUNTS (name, money, iban) VALUES (\"%s\", %f, \"%s\")", name, money, iban));
```
Comment (jantpedraza) [inline src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java]: Try here to use the `jdbcTemplate.update("query with ?", params)`.

**Review by jantpedraza — CHANGES_REQUESTED** [review]

File: src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java (hunk 1 above)
Comment (thok54) [reply src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java]: Done at last commit

File: src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java (hunk 2 above)
Comment (thok54) [reply src/main/java/com/banking/accountservice/repository/MySqlAccountRepository.java]: Done at last commit

**Review by jantpedraza — APPROVED** [review]

**Pull request merged**